src/hooks/usePermissions.js

```diff
@@ -0,0 +1,21 @@
+import _ from 'underscore';
+import {useContext, useMemo} from 'react';
+import Permissions from '../libs/Permissions';
+import {BetasContext} from '../components/OnyxProvider';
+
+export default function usePermissions() {
+    const betas = useContext(BetasContext);
+    return useMemo(
+        () =>
+            _.reduce(
+                Permissions,
+                (memo, checkerFunction, beta) => {
+                    // eslint-disable-next-line no-param-reassign
+                    memo[beta] = checkerFunction(betas);
+                    return memo;
+                },
+                {},
+            ),
```

Comment:
I thought about that, but it would be a lot more verbose, and anyone adding or removing betas would also have to remember to update the hook. This way, you just have to add a new function to `libs/Permissions` and the hook will automatically handle that. This seems more robust to me.